dojo/1.11/dojo.d.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference path="index.d.ts" />
-/// <reference path="../../doh/1.11/doh" />
+/// <reference path="../../doh/1.11/doh.d.ts" />
```

Comment:
Typescript should resolve `doh` or `doh.d.ts`... I guess we should be consistent though.